docs/development/build-instructions-osx.md

```diff
@@ -5,14 +5,34 @@ Follow the guidelines below for building Electron on macOS.
 ## Prerequisites
 
 * macOS >= 10.8
-* [Xcode](https://developer.apple.com/technologies/tools/) >= 5.1
+* [Xcode](https://developer.apple.com/technologies/tools/) >= 7.3.1
 * [node.js](http://nodejs.org) (external)
 
 If you are using the Python downloaded by Homebrew, you also need to install
 the following Python modules:
 
 * [pyobjc](https://pythonhosted.org/pyobjc/install.html)
 
+## macOS SDK
+
+If you're simply developing Electron and don't plan to redistribute your
+custom Electron build, you may skip this section.
+
+For certain features (e.g. pinch-zoom) to work properly, you must target the
+macOS 10.10 SDK.
+
+Official Electron builds are built with [Xcode 7.3.1](http://adcdownload.apple.com/Developer_Tools/Xcode_7.3.1/Xcode_7.3.1.dmg), which does not contain
```

Comment:
I was getting the following error building with 10.10 SDK on 7.3.1 on 10.12:

```
xcodebuild[65085:306926] [MT] DVTSDK: Skipped SDK /Applications/Xcode731.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.sdk; its version (10.10) is below required minimum (10.11) for the macosx platform.
```

Had to edit the `Info.plis` mentioned here: https://monorail-prod.appspot.com/p/chromium/issues/detail?id=649952#c1